tests/fiaas_deploy_daemon/deployer/kubernetes/test_adapter.py

```diff
@@ -51,6 +52,10 @@ def ingress_deployer(self):
     def autoscaler_deployer(self):
         return mock.create_autospec(AutoscalerDeployer)
 
+    @pytest.fixture(autouse=True)
+    def pod_disruption_budget_deployer(self):
```

Comment:
nitpick; It would be nice to add a test verifying that this mock is called on k8s.deploy, similar to the other deployers